### ISSUE TITLE
Where is the distance feature and orientation feature in the code?

### ISSUE THREAD

**Issue by puhuajiang** [YIN95/Group-Behavior-Recognition#1]
Hi, thanks for your awesome job!

In the paper, you said that you have used the distance feature and the orientation feature, but I did not find them in the code. Can you tell me how did you use the distance feature and the orientation feature?

thanks

**Comment by YIN95** [comment]
> Hi, thanks for your awesome job!
> 
> In the paper, you said that you have used the distance feature and the orientation feature, but I did not find them in the code. Can you tell me how did you use the distance feature and the orientation feature?
> 
> thanks

Hi, the distance features and the orientation features are concatenated to the output of each MST-GCN. 

The code is in another branch:

groupBodyOutput = torch.cat([groupBodyOutput, dist[i]], dim=1)
groupBodyOutput = torch.cat([groupBodyOutput, ang[i]], dim=1)
